src/ifElse.test.js

```diff
@@ -1,11 +1,30 @@
 'use strict';
 
+const { ifElse } = require("./ifElse");
+
 describe('ifElse', () => {
-  // const { ifElse } = require('./ifElse');
 
-  it('should ', () => {
+  it('condition = true -> run first callback', () => {
```

Comment:
test description should be understandable, try to rephrase this summary and for the second test

Reply:
fixed